common/src/main/java/com/microsoft/identity/common/internal/cache/MicrosoftFamilyOAuth2TokenCache.java

```diff
@@ -125,9 +130,27 @@ public ICacheRecord loadByFamilyId(@Nullable final String clientId,
             }
         }
 
+        if (null != target) {
+            for (final Credential credential : getAccountCredentialCache().getCredentials()) {
+                if (credential instanceof AccessTokenRecord) {
+                    final AccessTokenRecord atRecord = (AccessTokenRecord) credential;
+
+                    if (null != clientId && clientId.equals(atRecord.getClientId())
```

Comment:
This entire boolean condition could be refactored into a separate boolean method

Reply:
I'll elect not to take this change based on a few points:

- Increases method count
- You still need to read the method to determine how the check works
- Little potential for reuse